generator/config/accumulator/firstN.yaml

```diff
@@ -24,99 +24,122 @@ arguments:
             A positive integral expression that is either a constant or depends on the _id value for $group.
 tests:
     -
-        name: 'Null and Missing Values'
-        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/firstN/#null-and-missing-values'
```

Comment:
> Removed a few tests that no longer seem to point to anything in the docs - e.g. "firstN:Null and Missing Values".

This example is in the "Behavior" part of the docs.

https://www.mongodb.com/docs/manual/reference/operator/aggregation/firstN/#null-and-missing-values